lua/dooing/ui/windows/help.lua

```diff
@@ -0,0 +1,132 @@
+local M = {}
+
+-- Dependencies
+local config = require("dooing.config")
+
+-- Local window IDs and buffers
+local win_id = nil
+local buf_id = nil
+
+-- Namespace for highlighting
+local ns_id = vim.api.nvim_create_namespace("dooing_help")
+
+-- Content configuration
+local HELP_CONTENT = {
+	" Main window:",
+	" i           - Add new to-do",
+	" x           - Toggle to-do status",
+	" d           - Delete current to-do",
+	" D           - Delete all completed todos",
+	" q           - Close window",
+	" H           - Add due date to to-do ",
+	" r           - Remove to-do due date",
+	" T           - Add time estimation",
+	" R           - Remove time estimation",
+	" ?           - Toggle this help window",
+	" t           - Toggle tags window",
+	" c           - Clear active tag filter",
+	" e           - Edit to-do item",
+	" u           - Undo deletition",
+	" /           - Search todos",
+	" I           - Import todos",
+	" E           - Export todos",
+	" <leader>D   - Remove duplicates",
+	" ",
+	" Tags window:",
+	" e     - Edit tag",
+	" d     - Delete tag",
+	" <CR>  - Filter by tag",
+	" q     - Close window",
+	" ",
```

Comment:
Same comment here than in the `main.lua` file. Loading keybindings from the config will ensure the help buffer is always accurate